### PR TITLE
Added .travis.yml

### LINKED ISSUE #22: Set up Travis CI
This would help us in maintaining code quality and ensuring stability.

### PR DESCRIPTION
**Pull request by Monal5031**
Closes #22

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+services:
+  - postgresql
+env:
+  -DJANGO=1.10 DB=postgresql
+install:
+  - pip install -r requirements.txt
+before_script:
+  - sudo su - postgres
+  - psql
+  - create database odyssy;
```
Comment (aviaryan) [inline .travis.yml]: Commands are run by `bash` in `before_script`. So psql commands won't work. I recommend having a look at https://github.com/anyaudio/anyaudio-server/blob/master/.travis.yml

Comment (Monal5031) [inline .travis.yml]: Done sir 👍

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  -DJANGO=1.10 DB=postgresql
+install:
+  - pip install -r requirements.txt
+before_script:
+  - psql -c "CREATE DATABASE odyssy;" -U postgres
+  - psql -c "CREATE USER odyssy WITH PASSWORD 'odyssy';" -U postgres
+  - psql -c "GRANT ALL ON DATABASE odyssy TO odyssy;" -U postgres
+
+script:
+  - cd wsgi/odyssy
```
Comment (Monal5031) [inline .travis.yml]: should it be `cd wsgi/odyssy/` or this is one is correct?

Comment (aviaryan) [inline .travis.yml]: This will work.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+install:
+  - pip install -r requirements.txt
+before_script:
+  - psql -c "CREATE USER odyssy WITH CREATEUSER PASSWORD 'odyssy';" -U postgres
+  - psql -c "ALTER USER odyssy CREATEDB;" -U postgres
+  - psql -c "CREATE DATABASE odyssy;" -U postgres
+  - psql -c "GRANT ALL ON DATABASE odyssy TO odyssy;" -U postgres
+
+script:
+  - cd wsgi/odyssy && python manage.py migrate
```
Comment (singhpratyush) [inline .travis.yml]: `makemigrations` before `migrate`?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+  - 2.7
+  - 3.4
```
Comment (singhpratyush) [inline .travis.yml]: You may remove this as we are not supporting `python3` here.

Comment (Monal5031) [inline .travis.yml]: Okay I'll remove `python3`.

Comment (Monal5031) [inline .travis.yml]: @singhpratyush Thanks for help 😄

File: .travis.yml (hunk 1 above)
Comment (Monal5031) [inline .travis.yml]: @aviaryan Thanks for help 😄

**Pull request merged**